src/Features/CSharp/Portable/MakeClassAbstract/CSharpMakeClassAbstractCodeFixProvider.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.MakeClassAbstract;
+
+namespace Microsoft.CodeAnalysis.CSharp.MakeClassAbstract
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(CSharpMakeClassAbstractCodeFixProvider)), Shared]
+    internal sealed class CSharpMakeClassAbstractCodeFixProvider : AbstractMakeClassAbstractCodeFixProvider<ClassDeclarationSyntax>
+    {
+        public override ImmutableArray<string> FixableDiagnosticIds { get; } =
+               ImmutableArray.Create(
+                   "CS0513" // 'C.M()' is abstract but it is contained in non-abstract class 'C'
+               );
+
+        protected override bool IsValidMemberNode(SyntaxNode node)
```

Comment:
I have just noticed that `node` could be `null` and this is not handled. I will push a separate commit for this.